tsconfig.json

```diff
@@ -36,5 +36,8 @@
     "core/tests/karma.conf.ts"
   ],
   "include": ["core", "extensions", "typings"],
-  "exclude": ["core/tests"]
+  "exclude": [
+    "core/tests",
+    "extensions/classifiers/proto"
```

Comment:
Can we add this exclude to _tsconfig-strict.ts_ too?

Reply:
Done